Dockerfile

```diff
@@ -10,11 +10,22 @@ RUN set -ex && \
     rm -f /usr/libexec/gcc/x86_64-alpine-linux-musl/6.4.0/lto-wrapper && \
     rm -f /usr/bin/x86_64-alpine-linux-musl-gcj
 
+RUN set -ex && \
+    apk add --no-cache gcc g++ musl-dev python3 openjdk17 iptables ip6tables && \
+    apk add --no-cache --virtual .build-deps bash curl git && \
+    curl -O https://dl.google.com/go/go1.21.6.linux-amd64.tar.gz && \
+    tar -C /usr/local -xzf go1.21.6.linux-amd64.tar.gz && \
+    rm go1.21.6.linux-amd64.tar.gz && \
+    apk del .build-deps
+
 RUN ln -sf python3 /usr/bin/python
 
 ADD . /usr/bin/
 ADD start.sh /usr/bin/
 
+ENV PATH=$PATH:/usr/local/go/bin
+ENV GOPATH=/go
```

Comment:
This is also not needed.

Reply:
without 
`ENV PATH=$PATH:/usr/local/go/bin`
this line go lang compiler is not recognized by doker.